reference/api/indexes.md

```diff
@@ -44,15 +44,15 @@ List all [indexes](/learn/core_concepts/indexes.md).
 
 ## Get one index
 
-<RouteHighlighter method="GET" route="/indexes/{uid}"/>
+<RouteHighlighter method="GET" route="/indexes/{index_uid}"/>
```

Comment:
Agree with the different `uid`s being confusing, but `index_uid` doesn't technically exist, so is it okay to use it here?

Reply:
Maybe we should go with `index_uid` for consistency. We have used this format for [dumps](https://docs.meilisearch.com/reference/api/dump.html#get-dump-status) and indexes at different places:
- [Search](https://docs.meilisearch.com/reference/api/search.html#search-in-an-index-with-get-route) 
- [Documents](https://docs.meilisearch.com/reference/api/documents.html#get-documents)
- [Settings](https://docs.meilisearch.com/reference/api/displayed_attributes.html#get-displayed-attributes)